starter-code/index.html

```diff
@@ -0,0 +1,206 @@
+              <a class="dropdown-item" href="#">Full page</a>
+              <a class="dropdown-item" href="#">Child page 1</a>
+              <a class="dropdown-item" href="#">Child Page 2</a>
+            </div>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link my-highlight" href="#">Contact</a>
+          </li>
+        </ul>
+      </div>
+    </nav>
+  </div>
+  <!-- end top header -->
+
+  <!-- slidebox container -->
+  <!-- temporary placeholder -->
+  <div class="slidebox">
+
+  </div>
+  <!-- end slidebox container -->
+
+  <!-- fidgets -->
+  <div class="fidgets">
+      <div class="container">
+          <div class="row">
+              <div class="col-md-4 my-fidget">
+                <h1>Web design</h1>
+                <p>Cum sociis natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus. Vestibulum vehicula erat non congue cursus. Aenean auctor nulla quis augue dictum, sed sagittis odio varius <a id="black-hover" href="#">Link</a></p>
+              </div>
+              <div class="col-md-4 my-fidget">
+                <h1>Web development</h1>
+                <p>Duis tempus leo vitae ipsum viverra, blandit condimentum sapien porttitor. Duis porttitor sed metus eget mollis. Curabitur bibendum imperdiet tortor, ut pulvinar purus elementum nspendisse at erat luctus.</p>
+              </div>
+              <div class="col-md-4 my-fidget">
+                <h1>User interface</h1>
+                <p>Cras aliquet venenatis porttitor. Donec rutrum sapien et sapien dignissim, eu dapibus mi molestie. Suspendisse dictum convallis quam et sodales. Praesent non enim et magna congue gravida.</p>
+              </div>
+          </div>
+          <div class="row"></div>
+      </div>
+  </div>
+  <!-- end fidgets -->
+
+  <!-- Latest Project -->
+  <div class="container">
+      <div class="row">
+          <div class="col-md-4">
+            <h3>LATEST PROJECTS</h3>
+            <h4>FEW OF THE LATEST PORTFOLIO ITEMS</h4>
+          </div>
+      </div>
+      <div class="row">
+          <div class="col-md-3">
+            <img class="my-image" src="img/project-1.jpg" alt="">
+            <p>Nunc libero elit, iaculis nec augue non</p>
+          </div>
+          <div class="col-md-3">
+            <img class="my-image" src="img/project-2.jpg" alt="">
+            <p>Phasellus aliquet iaculis neque accumsan aliquam</p>
+          </div>
+          <div class="col-md-3">
+            <img class="my-image" src="img/project-3.jpg" alt="">
+            <p>Vivamus blandit ipsum sed arcu euismod</p>
+          </div>
+          <div class="col-md-3">
+            <img class="my-image" src="img/project-4.jpg" alt="">
+            <p>Curabitur auctor dolor risus, vitae pellentesque arcu consequat</p>
+          </div>
+      </div>
+  </div>
+
+  <!-- end Latest Projects -->
+
+
+  <!-- latest articles -->
+
+  <div class="container">
```

Comment:
needed a bit more spacing between the top div and this one. You can always just do `<br>` to add spacing or an empty div with a height